structs.go

```diff
@@ -304,16 +304,16 @@ type Game struct {
 // UnmarshalJSON unmarshals json to Game struct
 func (g *Game) UnmarshalJSON(bytes []byte) error {
 	temp := &struct {
-		Name string          `json:"name"`
+		Name json.Number     `json:"name"`
```

Comment:
![image](https://cloud.githubusercontent.com/assets/1083807/25810381/3a349308-3410-11e7-8840-181dc19d3134.png)
it won't fail